README.md

```diff
@@ -3,9 +3,42 @@
     <img src="https://raw.githubusercontent.com/ashleymcnamara/gophers/296b4d47f5313822b348e442837ca2d32a7704a3/Azure_Gophers.png" width="360"></a>
 </p>
 
-# cerulean
-Cerulean is a golang library for mocking the [azure-sdk-for-go](https://github.com/Azure/azure-sdk-for-go).
+# Cerulean
 
+[![WIP](https://img.shields.io/badge/alpha-unstable-yellow])]() [![Go Report Card](https://goreportcard.com/badge/github.com/goshlanguage/cerulean)](https://goreportcard.com/report/github.com/goshlanguage/cerulean) [![Man Hours](https://img.shields.io/endpoint?url=https%3A%2F%2Fmh.jessemillar.com%2Fhours%3Frepo%3Dhttps%3A%2F%2Fgithub.com%2Fgoshlanguage%2Fcerulean.git)](https://jessemillar.com/r/man-hours)
+
+Cerulean is a mock Azure API, designed for writing infrastructure tests so you can run tests without spending that `dosh`. This is great for folks who are using the `azure-sdk-for-go` especially. Here's a nifty example:
+
+```go
+package test
+
+import (
+    "fmt"
+    "testing"
+
+    "github.com/azure/azure-sdk-for-go"
+    "github.com/goshlanguage/cerulean"
+)
+
+func TestSubscription(t *testing.T) {
+    server := cerulean.New(":8080", "c27e7a81-b684-4fce-91d8-fed9e9bb534a")
+}
+```
+
+This project is artisanally crafted, laptop to market software. Please open an issue if you find a bug or have a request. PRs are welcomeß.
```

Comment:
Just noticed when I was adding some badges to the README just now. Did you mean to have a Greek character at the very end of this sentence? I left it in case it's a joke/reference I just don't get.